cylc/flow/task_events_mgr.py

```diff
@@ -972,22 +975,20 @@ def _setup_event_mail(self, itask, event):
         if (id_key in self.event_timers or
                 event not in self._get_events_conf(itask, "mail events", [])):
             return
-        retry_delays = self._get_events_conf(itask, "mail retry delays")
-        if not retry_delays:
-            retry_delays = [0]
+
         self.event_timers[id_key] = TaskActionTimer(
```

Comment:
Do we still need a TaskActionTimer now that we have removed the retry delays?

Reply:
It breaks things if it's not there - but I would have thought so - I think that the `TaskEventsMgr.event_times` expects values to be `TaskActionTimer` objects.